lib/clearsale/authentication.rb

```diff
@@ -0,0 +1,48 @@
+require 'rubygems'
+require 'multi_json'
+require 'json'
+
+module Clearsale
+  class Authentication
+    def self.connector
+      @connector = Connector.new('auth/login')
+    end
+
+    def self.request_auth
```

Comment:
You should do something to store the authentication token on a class attribute so that you only make requests for the token when you need it and not on every single request.